README.md

```diff
@@ -538,55 +497,51 @@ If you acknowledge the warning above and are still looking to try a fork that su
         <strong>Long:</strong> Yes from community (from chrispypatt) <br/> <br/>
         <ul>
-          <li>Pre-built version of `sunnypilot/master-new`</li>
-          <li>Model switcher to easily switch between various models</li>
-          <li>NNLC: Big steering improvements for '21-23 RAV4 Prime and Sienna</li>
-          <li>Ships the same latest improvements as nightly-dev (with the same risk of breaking on rare occasions).</li>
+          <li>First fork to get long!</li>
+          <li>Very quick stop-and-go response</li>
+          <li><code>alexandresato/extract_secoc_key_btn</code> includes a TSK key extract button and is rebased with <code>personal3</code> often.</li>
         </ul>
       </td>
     </tr>
     <tr>
       <td>
-        <code>chrispypatt/frogpilot-r4p</code> <br/>
-        (fork of FrogPilot) <br/>
-        <a href="https://discord.com/invite/frogpilot">FrogPilot discord</a>
+        <code>install.sunnypilot.ai/staging</code><br/>
+        (a.k.a. sunnypilot)<br/>
+        <a href="https://discord.gg/sunnypilot">sunnypilot discord</a>
       </td>
       <td>
         <strong>Lat:</strong> Yes from upstream <br/>
-        <strong>MADS:</strong> Yes from community (AOL from FP) <br/>
-        <strong>Long:</strong> Yes from community (original author) <br/> <br/>
+        <strong>MADS:</strong> Yes from community (MADS original author) <br/>
+        <strong>Long:</strong> Yes from community (from chrispypatt) <br/> <br/>
         <ul>
-          <li>chrispypatt's fork with TSK long support.</li>
-          <li>Uses an old AGNOS version. When downgrading, OP may get stuck in a registration loop. In this case, tap-tap-tap on the boot logo and reset the device to recover and then install again.</li>
-          <li>‼️ DO NOT RUN FROGPILOT DEEP STORAGE DELETE. It deletes your security key, and you have to run TSK Manager again. Run it only when you're selling the device.</li>
+          <li>Pre-built version of `sunnypilot/master`</li>
+          <li>Model switcher to easily switch between various models</li>
+          <li>NNLC: Big steering improvements for '21-23 RAV4 Prime and Sienna</li>
+          <li>Ships the same latest improvements as nightly-dev (with the same risk of breaking on rare occasions).</li>
         </ul>
       </td>
     </tr>
     <tr>
       <td>
-        <code>optskug/SiennaFP</code> <br/>
-        (fork of FrogPilot) <br/>
+        <code>frogpilot.download</code> <br/>
+        (a.k.a. FrogPilot)<br/>
         <a href="https://discord.com/invite/frogpilot">FrogPilot discord</a>
       </td>
       <td>
-        <strong>Lat:</strong> Yes from community (from anrum) <br/>
-        <strong>MADS:</strong> Yes from community (AOL from FP) <br/>
-        <strong>Long:</strong> No <br/> <br/>
+        <strong>Lat:</strong> Yes from upstream <br/>
+        <strong>MADS:</strong> Yes from community (AOL original author) <br/>
```

Comment:
Is this accurate? I feel like we dug into this before and got an answer before...